src/locales/en.json

```diff
@@ -1941,6 +1962,7 @@
 "label.tuesday": "Tuesday",
 "label.type": "Type",
 "label.type.id": "Type ID",
+"label.tx.size": "TX {number}",
```

Comment:
@utchoang don't use this synyax, it will be difficult for our translators, please keep it simply text strings without code, instead you can add variable to translations in components

Reply:
@rhtyd Can I change it to an image? Or just allow string concatenation in the component.
![image](https://user-images.githubusercontent.com/13766648/86324397-bd7e1980-bc68-11ea-8a52-1bc510b6b27e.png)

Reply:
Ah. Sorry, I was misunderstood. You mean don't use the syntax (Exp: TX, SR,...)

Reply:
Yes don't use the `{}` braces in translation strings